src/generation_tools.jl

```diff
@@ -1,5 +1,5 @@
 ## generation_tools.jl
-export ϕ_tuple, idx_to_dim
+export ϕ_tuple, idx_to_dim, dim_to_idx, idx_to_arr_view, arr_to_idx_view
```

Comment:
I would not export any of those function, in fact I am totally against them -> seel below

Reply:
I removed those exports